xdocs/usermanual/get-started.xml

```diff
@@ -279,7 +279,7 @@ jmeter -t test.jmx &hellip;
 <dl>
 <dt><code>DDRAW</code></dt><dd>JVM options to influence usage of direct draw, e.g. <code>-Dsun.java2d.ddscale=true</code>. Default is empty.</dd>
 <dt><code>GC_ALGO</code></dt><dd>JVM garbage collector options. Defaults to <code>-XX:+UseG1GC -XX:MaxGCPauseMillis=250 -XX:G1ReservePercent=20</code></dd>
-<dt><code>HEAP</code></dt><dd>JVM memory settings used when starting JMeter. Defaults to <code>-Xms1g -Xmx1g -XX:MaxMetaspaceSize=256m</code></dd>
+<dt><code>HEAP</code></dt><dd>JVM memory settings used when starting JMeter. Defaults to <code>-Xms1g -Xmx1g -X:MaxMetaspaceSize=256m</code></dd>
```

Comment:
Could you please clarify the reason for the chnage? AFAIK, `-XX:MaxMetaspaceSize=256m` is the proper syntax

Reply:
I guess it's a typo

Reply:
`-XX:MaxMetaspaceSize=256m` and `-X:MaxMetaspaceSize=256m` that is correct?

Reply:
As @vlsi said, the documentation currently shows the correct option. The correct one is `-XX:MaxMetaspaceSize=...`.